templates/common/ovirt/units/qemu-guest-agent.service.yaml

```diff
@@ -0,0 +1,34 @@
+name: qemu-guest-agent.service
+enabled: true
+contents: |
+  {{ if .Infra -}}
+  {{ if .Infra.Status -}}
+  {{ if .Infra.Status.PlatformStatus -}}
+  
+  [Unit]
+  Description=qemu guest agent container starter
+  Wants=network-online.target
+  After=network-online.target ignition-firstboot-complete.service
+  Before=kubelet.service crio.service
+
+  [Service]
+  Type=simple
+  ExecStart=/usr/bin/podman run --rm \
+    --name qemu-guest-agent \
+    --privileged \
+    --volume /dev:/dev \
+    --volume /dev/virtio-ports:/dev/virtio-ports \
+    --volume /etc:/etc \
+    --volume /run/dbus:/run/dbus \
+    --volume /run/systemd:/run/systemd  \
+    --volume /usr/share/zoneinfo:/usr/share/zoneinfo \
+    --net=host \
+    --pull missing \
+    docker://registry.svc.ci.openshift.org/ovirt/qemu-guest-agent:4.2
```

Comment:
We cannot pull arbitrary container images with OpenShift 4 by default.  Anything we run by default must be part of the release image.

Reply:
And once it's part of the release image, this would need to have its value substituted by templating, same as other containers the MCO references.

Reply:
agreed, we build this container in ovirt NS just for testing purposes, we want to see it running and docume